Makefile.toml

```diff
@@ -0,0 +1,98 @@
+[tasks.frontend]
+command = "./client/build"
+toolchain = "nightly"
+workspace = false
+dependencies = [ "atomify" ]
+
+[tasks.serve-frontend]
+script = [
+'''
+#!/usr/bin/env bash
+
+browser=$(command -v chromium 2>/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Please install chromium for the frontend as it's the only browser with offscreen canvas support!"
+  exit 1
+fi
+
+top=$(git rev-parse --show-toplevel 2>/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Please run this script from the ratatosk git repository!"
+  exit 1
+fi
+cd $top/client
+
+trap 'kill -TERM "$pid" 2>/dev/null' TERM INT
+python3 -m http.server 3000 &
+pid=$!
+chromium http://localhost:3000
+
+wait $pid
+trap - TERM INT
+wait $pid
+EXIT_STATUS=$?
+'''
+]
+workspace = false
+dependencies = [ "frontend" ]
+
+[tasks.tools]
+workspace = false
+dependencies = [ "atomify", "bone_cake" ]
+
+[tasks.atomify]
+workspace = false
+toolchain = "nightly"
+command = "./tools/atomify/build"
+
+[tasks.bone_cake]
+workspace = false
+toolchain = "nightly"
+script = [
+'''
+#!/usr/bin/env bash
+cd tools/bone_cake # TODO refactor
+cargo build
```

Comment:
I strongly dislike this hack. With `--cwd` you can alter the directory for `cargo-make` from the CLI which would break this. I'll check if there's a declarative way to switch dirs (maybe with sub-makes like `make -C` with GNU/Make), if that's not possible I'd write a simple bash helper for `git rev-parse --show-toplevel` that is used everywhere :)

Reply:
It's possible to use `cargo <cmd> -p` which is way more straightforward.